packages/sdk/contractkit/src/wrappers/SortedOracles.ts

```diff
@@ -46,53 +49,94 @@ export interface MedianRate {
   rate: BigNumber
 }
 
+export type CurrencyPairIdentifier = Branded<Address, 'PairIdentifier'>
+
+/**
+ * Used to construct the pair identifier from a pair label (e.g. CELO/BTC)
+ * The pair identifier needs to be a valid ethereum address, thus we
+ * truncate a keccak of the pair label.
+ * This function returns a branded type which can be fed into the wrapper.
+ * @param pair a string
+ */
+export const pairIdentifier = (pair: string): CurrencyPairIdentifier => {
+  return ensureLeading0x(
+    keccak256(pair)
+      .slice(0, 20)
+      .toString('hex')
+  ) as CurrencyPairIdentifier
+}
+
+/**
+ * This will act as an enum of common pairs.
+ * We can't use a straight enum because we want the value
+ * to be the branded type.
+ *
+ * E.g. usage: sortedOracles.getRates(CurrencyPair.CELOBTC)
+ */
+type defaultPairs = 'CELOBTC' | 'CELOETH'
+export const CurrencyPair: Record<defaultPairs, CurrencyPairIdentifier> = {
+  CELOBTC: pairIdentifier('CELO/BTC'),
+  CELOETH: pairIdentifier('CELO/ETH'),
+}
```

Comment:
A few nits around this:

1. Shall we remove `CELOETH` as it could led people think rates for this pair are available in the Oracles?
2. Related to the above, the name `CurrencyPair` feels a bit broad to me, maybe something more explicit around currency pairs available in sortedOracles would be nicer? Maybe it's just me though, so fine to leave it as it is if others don't have an issue with it. 

And lastly a bit of a random thought but I was thinking that at some point it would be nice to have `CELOCUSD` as part of the enum mapped to `CeloContract.StableToken`, so one can find all the pairs in a single place.

Reply:
Or even `CELOUSD`, since that's actually what the reported values are.

Reply:
Oh yes sorry, totally meant `CELOUSD` to not go back to the current confusing situation :P

Reply:
I like that idea around CELOUSD! 

But about CurrencyPair being a bit broad, do we have any other name suggestion, I'm spacing.

Reply:
I can only think of `SortedOraclesCurrencyPair` or just `SortedOraclesPair`, so any suggestions welcomed!

Reply:
Maybe just `OracleCurrencyPair` I think the "Sorted" part is a bit superfluous in this context, but just "Pair" without "Currency" sounds weird. Wdyt?

Reply:
Sounds great to me! 🚢